src/components/content/systemStatus/useSystemStatusQuery.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { useQuery } from '@tanstack/react-query';
+import { AdminService } from '../../../xpanse-api/generated';
+import { systemStatusQueryKey } from './queryKey';
+
+export function useSystemStatusQuery() {
```

Comment:
I just realized that the method name has changed to HealthCheck. So please rename the hook accordingly.